.github/workflows/release.yaml

```diff
@@ -145,9 +131,9 @@ jobs:
         run: |
           # Retry 3 times before the steps actually fails
           (echo "===== Maven Deploy Attempt: 1 ====" && mvn deploy -Prelease --batch-mode --settings /home/runner/.m2/settings.xml -DskipTests) || \
-          (echo "===== Maven Deploy Attempt 1 Failed - Sleeping for 30 minutes ====" && sleep 30m) || \
```

Comment:
fix for the 'false positive'. Use **&&** instead of **||**

Reply:
Unfortunately this change seems to have resulted in the maven deploy step running again even when it succeeds...resulting in an overall failure because the deploy is rejected the 2nd time (because it was already done).